repose-aggregator/extensions/simple-rbac/src/main/scala/org/openrepose/filters/simplerbac/SimpleRbacFilter.scala

```diff
@@ -0,0 +1,261 @@
+      val mutableHttpResponse = MutableHttpServletResponse.wrap(mutableHttpRequest, servletResponse.asInstanceOf[HttpServletResponse])
+
+      logger.trace("Simple RBAC filter processing request...")
+      validator.validate(mutableHttpRequest,mutableHttpResponse,filterChain)
+    }
+    logger.trace("Simple RBAC filter returning response...")
+  }
+
+  override def configurationUpdated(configurationObject: SimpleRbacConfig): Unit = {
+    configuration = configurationObject
+    config.enableRaxRolesExtension = true
+    config.checkPlainParams = true
+    config.maskRaxRoles403 = configuration.isEnableMasking403S
+    config.setResultHandler(getHandlers)
+
+    val rbacWadl = rbacToWadl(Option(configuration.getResources)).orElse(
+      Option(configuration.getResourcesFileName: String) match {
+        case Some(n) =>
+          rbacToWadl(readResource(
+              configurationService.getResourceResolver.resolve(n).newInputStream()
+          ))
+        case _ =>
+          None
+      }
+    )
+    rbacWadl match {
+      case Some(r) =>
+        Option(configuration.getWadlOutput: String) match {
+          case Some(wadlOutput) =>
+            val wadlPath: Path = Paths.get(StringUriUtilities.formatUri(getPath(wadlOutput, configurationRoot))).toAbsolutePath
+            try {
+              if (Files.exists(wadlPath) && Files.isWritable(wadlPath) || !Files.exists(wadlPath) && Files.createFile(wadlPath) != null) {
+                Files.write(wadlPath, r.getBytes)
+                logger.debug("Saved WADL to: " + wadlPath)
+              } else {
+                logger.warn("Cannot write to WADL file: " + wadlPath)
+                logger.debug(s"Generated WADL:\n\n$r\n")
+              }
+            } catch {
+              case ex: IOException =>
+                logger.warn("Cannot write to WADL file: " + wadlPath, ex)
+                logger.debug(s"Generated WADL:\n\n$r\n")
+            }
+          case _ =>
+            logger.debug(s"Generated WADL:\n\n$r\n")
+        }
+        val uuid = UUID.randomUUID
+        validator = Validator.apply(
+          s"SimpleRbacValidator_$uuid",
+          new StreamSource(new ByteArrayInputStream(r.getBytes), "file://simple-rbac.wadl"),
+          config
+        )
+        initialized = true
+      case _ =>
+        logger.warn("Unable to generate the WADL; check the provided resources.")
+    }
+  }
+
+  override def isInitialized: Boolean = initialized
+
+  private def getHandlers: DispatchHandler = {
+    val handlers: util.List[ResultHandler] = new util.ArrayList[ResultHandler]
+    Option(configuration.getDelegating) match {
+      case Some(d) =>
+        handlers.add(new MethodLabelHandler)
+        handlers.add(new DelegationHandler(d.getQuality))
+      case _ =>
+        handlers.add(new ServletResultHandler)
+    }
+    handlers.add(new InstrumentedHandler)
+    handlers.add(new ApiCoverageHandler)
```

Comment:
shouldn't this one be conditional? i thought it was a feature that had to be turned on in the other filter. is that not the case?